engine/src/main/java/org/camunda/bpm/engine/impl/core/operation/AbstractEventAtomicOperation.java

```diff
@@ -114,4 +115,8 @@ protected void eventNotificationsFailed(T execution, Exception exception) {
       throw new PvmException("couldn't execute event listener : " + exception.getMessage(), exception);
     }
   }
+
+  protected void setDelayedPayloadToNewScope(T execution, CoreModelElement scope) {
```

Comment:
💬 For me this method seems very specific in the abstract class and it's only used in one atomic operation. Some thoughts:
- Theoretically the implementation in `PvmAtomicOperationActivityStart` could also be part of its `getScope` method.
- We could rename it to be something more generic, `initNewScope`.

I'm also fine with this, but curious to hear opinions.